sonic_cli_gen/yang_parser.py

```diff
@@ -401,10 +402,10 @@ def get_mandatory(y_leaf: OrderedDict) -> bool:
             'leaf' 'mandatory' value
     """
 
-    if y_leaf.get('mandatory') is not None:
-        return True
+    if y_leaf.get('mandatory') is None:
```

Comment:
>None [](http://example.com/codeflow?start=34&length=4)

Thanks for this bug fix! Could you separate it into a standalone PR?

Reply:
Logic of both code is same. 
Though, assume  we have to wrap it in try-catch. In that case, what should be return value from catch block i.e. in exception case? 

-- If false then current block is good.
-- If true then new block is good.

I prefer, false, because by default, each node is not mandatory.